rtbkit/common/exchange_connector.cc

```diff
@@ -121,6 +121,21 @@ getBidSourceConfiguration() const
     return "{\"type\":\"unknown\"}";
 }
 
+bool
+ExchangeConnector::
+getBidValidity (const Bid  & bid,
+               const std::vector<AdSpot> & imp,
+               int & spotIndex) const
```

Comment:
I would avoid passing simple primitive types like `int` by reference. By value is perfectly fine.